attackers/client.py

```diff
@@ -0,0 +1,50 @@
+import socket
+import response_handlers
+from create_update_request import update_request
+from common.const import *
+from common import protocol
+
+
+def connect_to_server(address: ADDR) -> socket.socket:
+    my_socket = socket.socket()
+    my_socket.connect(address)
+    return my_socket
+
+
+def create_request() -> FULL_REQUEST:
+    cmd = input(CMD_INPUT).lower()
+    if cmd == "update":
+        params = update_request()
+    else:
+        params = [input(PARAMS_INPUT).lower()]
+    return cmd, params
+
+
+def handle_response(response_cmd: str, response_params: str):
+    handler_function = getattr(response_handlers, response_cmd)
+    handler_function(response_params)
+    print("-" * 146)
+
+
+def is_client_disconnect(cmd: str) -> bool:
+    return cmd != "disconnect"
+
+
+def main():
+    my_socket, is_connect = connect_to_server(SERVER_ADDRESS), True

```

Comment:
No need to create a tuple and unpack it right away.   
```python
is_connect = True
my_socket = connect_to_server(SERVER_ADDRESS)
```   
This is much more readable